src/Features/CSharp/Portable/ConvertToRawString/ConvertRegularStringToRawStringCodeRefactoringProvider.cs

```diff
@@ -159,6 +209,52 @@ private static bool HasLeadingWhitespace(VirtualCharSequence characters)
             return document.WithSyntaxRoot(root.ReplaceToken(token, replacement));
         }
 
+        protected override async Task FixAllAsync(
+            Document document,
+            ImmutableArray<TextSpan> fixAllSpans,
+            SyntaxEditor editor,
+            CodeActionOptionsProvider optionsProvider,
+            string? equivalenceKey,
+            CancellationToken cancellationToken)
+        {
+            Debug.Assert(equivalenceKey != null);
+
+            var options = await document.GetSyntaxFormattingOptionsAsync(optionsProvider, cancellationToken).ConfigureAwait(false);
+            using var _ = PooledDictionary<SyntaxToken, SyntaxToken>.GetInstance(out var tokenReplacementMap);
+
+            foreach (var fixSpan in fixAllSpans)
+            {
+                var node = editor.OriginalRoot.FindNode(fixSpan);
+                foreach (var stringLiteral in node.DescendantTokens(descendIntoTrivia: true).Where(token => token.Kind() == SyntaxKind.StringLiteralToken))
+                {
+                    // Ensure we can convert the string literal
+                    if (!CanConvertStringLiteral(stringLiteral, out var canConvertParams))
+                        continue;
+
+                    // Get the kind to be fixed from the equivalenceKey for the FixAll operation
+                    var kind = s_kindToEquivalenceKeyMap.GetKeyOrDefault(equivalenceKey);
+
+                    // Ensure we have a matching kind to fix for this literal
+                    var hasMatchingKind = kind switch
+                    {
+                        ConvertToRawKind.SingleLine => canConvertParams.CanBeSingleLine,
+                        ConvertToRawKind.MultiLineIndented => !canConvertParams.CanBeSingleLine,
+                        ConvertToRawKind.MultiLineWithoutLeadingWhitespace => canConvertParams.CanBeMultiLineWithoutLeadingWhiteSpaces,
```

Comment:
why not just have teh params include the kind?

Reply:
Because for the non-single line case, both `MultiLineIndented` and `MultiLineWithoutLeadingWhitespace` are valid kinds, and it might be confusing to expose an array of applicable kinds.